frontend/src/components/dashboard/page_timings/PageTimingsTable.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+    <VContainer
+        p-0
+        class="users-table"
+    >
+        <VRow
+            class="header my-3"
+            fluid
+        >
+            <VCol>
+                <VSelect
+                    class="option-select"
+                    prefix="Show"
+                    :items="options"
+                    flat
+                    :value="value"
+                    @change="changeSelect"
+                />
+            </VCol>
+            <VCol>
+                {{ label }}
+            </VCol>
+        </VRow>
+        <VDataTable
+            class="caption"
+            hide-default-footer
+            hide-default-header
+            :headers="headers"
+            :items="items"
+        />
+    </VContainer>
+</template>
+
+<script>
+    export default {
+        name: 'PageTimingsTable',
+        props: {
+            items: {
+                type: Array,
+                required: true
+            },
+            label: {
+                type: String,
+                required: true
+            },
+            value: {
+                type: String,
+                required: true
+            }
+        },
+        data () {
+            return {
+                options: [
+                    {
+                        text: 'Page',
+                        value: 'page'
+                    },
+                    {
+                        text: 'Browser',
+                        value: 'browser'
+                    },
+                    {
+                        text: 'Country',
+                        value: 'country'
+                    },
+                ],
+                headers: [
+                    { text: '', align: 'center', value: 'name' },
+                    { text: '', align: 'center', value: 'value' },
+                ],
+            };
+        },
+        methods: {
+            changeSelect () {
+                this.$emit('change', this.value);
```

Comment:
It won't work, you need to emit value, that this method receives, but the value, which you got from props